companion/index.js

```diff
@@ -27,7 +27,7 @@ settingsStorage.onchange = function(evt) {
 function getRequesttOptions () {
   const options = {};
 
-  if (settings.apiSecret) {
+  if (settings.apiSecret && !settings.offline) {
```

Comment:
This is not correct, xDrip does actually require the api secret if the xDrip server port is opened to local lan in addition to localhost

Reply:
According to this: https://github.com/NightscoutFoundation/xDrip/blob/master/Documentation/technical/Local_Web_Services.md
It does not require the API Secret.